lib/suse-cloud-upgrade-4-to-5-post

```diff
@@ -58,6 +58,17 @@ for barclamp in updater suse-manager-client nfs_client cisco-ucs hyperv heat cei
 done
 
 
+### Put all the cluster nodes back online
+for founder in $(<"${UPGRADE_DATA_DIR}/cluster_founders"); do
+  echo_summary "Putting cluster nodes online from founder $founder..."
+  ssh $founder '
+    for node in $(crm node list | grep -v standby | cut -d : -f 1); do
```

Comment:
Here we are running in the same problem I had yesterday, as a 'maintenance' string is also a possible outcome.